Documentation/config/postgres-database-cfg.yaml

```diff
@@ -0,0 +1,101 @@
+kind: List 
+metadata: {}
+apiVersion: v1
+items:
+  - apiVersion: image.openshift.io/v1
+    kind: ImageStream
+    metadata:
+      name: postgresql-10-rhel7
+    spec:
+      lookupPolicy:
+        local: true
+      tags:
+        - name: latest
+          from:
+            kind: DockerImage
+            name: registry.access.redhat.com/rhscl/postgresql-10-rhel7:latest
```

Comment:
We are doing table partitioning so we are using latest Postgres image - image: "quay.io/leihaochen/curator-postgresql-extension:13"